src/commands/ShutdownRoomCommand.ts

```diff
@@ -20,6 +20,7 @@ import { RichReply } from "matrix-bot-sdk";
 // !mjolnir shutdown room <room> [<message>]
 export async function execShutdownRoomCommand(roomId: string, event: any, mjolnir: Mjolnir, parts: string[]) {
     const victim = parts[3];
+    const reason = parts.slice(4).join(" ");
```

Comment:
Well I asked because some of our other things have `<no reason supplied>` as the default and the reason is user facing I believe, so it might be worth sticking to that

Reply:
https://github.com/matrix-org/mjolnir/blob/eb7f5f6b3e833b2000089451d2a57595045ec880/src/Mjolnir.ts#L1010